### PR TITLE
Add ORF metadata

### PR DESCRIPTION
**Pull request by shntnu**

### CODE REVIEW

**Comment by shntnu** [comment]
@niranjchandrasekaran can you help fill out these:

https://github.com/jump-cellpainting/datasets/pull/14/files#diff-d95c905065c7d84ac9b2f43ed6ece5149c6479cd442d55a9216de338004bdbdcR31-R41

File: metadata/README.md [hunk 1]
```diff
@@ -24,4 +24,21 @@ erDiagram
         string Metadata_InChI "International Chemical Identifier"
         string Metadata_InChIKey "Hashed InChI"
     }
+    WELL }o--o| ORF : ""
+    ORF {
+        string Metadata_JCP2022 PK "Perturbation identifier"
+        string Metadata_NCBI_Gene_ID "NCBI unique identifier of a gene"
+        string Metadata_broad_sample ""
+        string Metadata_Name ""
```
[the remark this answers is not in the document]
Comment (niranjchandrasekaran) [reply metadata/README.md:32]: I am not 100% certain about this

File: metadata/README.md [hunk 2]
```diff
@@ -24,4 +24,21 @@ erDiagram
...
+        string Metadata_Vector ""
+        string Metadata_Transcript ""
+        string Metadata_Symbol ""
+        string Metadata_Taxon_ID ""
+        string Metadata_Gene_Description ""
+        string Metadata_Annot_Gene_Symbol ""
```
Comment (niranjchandrasekaran) [inline metadata/README.md:38]:
```suggestion
        string Metadata_Annot_Gene_Symbol "NCBI RefSeq gene name"
```

File: metadata/README.md [hunk 3]
```diff
@@ -24,4 +24,21 @@ erDiagram
...
+        string Metadata_Annot_Gene_ID ""
```
Comment (niranjchandrasekaran) [inline metadata/README.md:39]:
```suggestion
        string Metadata_Annot_Gene_ID "NCBI RefSeq gene ID"
```

File: metadata/README.md [hunk 4]
```diff
@@ -24,4 +24,21 @@ erDiagram
...
+        string Metadata_Prot_Match ""
+        string Metadata_Insert_Length ""
```
Comment (niranjchandrasekaran) [inline metadata/README.md:41]: I can probably guess what these are but I don't know for sure.

Comment (shntnu) [reply metadata/README.md:41]: I think Metadata_Prot_Match = % match to protein sequence

Comment (shntnu) [reply metadata/README.md]: `Insert_Length`:

image

**Pull request merged**